### DISCUSSION TITLE
Is it possible to create a preset from many presets?

### DISCUSSION THREAD

**Discussion by jeco123** [tailwindlabs/tailwindcss#5307] [Help]
Hi,

I'm creating a design system tailwind preset to expose our design system to our teams. I got packages colors/fontSize/grids etc... and each package is a preset. Then i have my design system tailwind preset which import all my atomic presets.

By importing the design system tailwind preset in a tailwind config, it does not work. By importing all my atomic presets in my tailwind config, it does not work.

Is it a bug or is it normal?

Regards,

Jérôme

**Comment by huibizhang** [comment]
Trying just add your all preset files in `preset: []` in tailwind.config.js ?
